main.go

```diff
@@ -167,11 +172,11 @@ func Main() int {
 	if fd == -1 {
 		fd, tapMode, mtu, err = GetTunTap(netNsPath, ifName)
 		if err != nil {
-			panic(fmt.Sprintf("Failed to open TUN/TAP: %s", err))
+			failNow(fmt.Sprintf("Failed to open TUN/TAP: %s", err))
 		}
```

Comment:
I prefer explicit os.Exit() here. Yes, redundant. But we're not in perl with `die`, we're in golang and repeating stuff is the way to go.
Also, with explicit os.Exit it's more obivious if you decide to  use different exit codes.
Also, exiting from main() for me smells ok (you can always "return"!) and exiting from not-main is similar to panic. which is sad.